dts/bindings/flash_controller/nordic,nrf51-flash-controller.yaml

```diff
@@ -0,0 +1,19 @@
+---
+title: Nordic NVMC
+id: nordic,nrf-nvmc
```

Comment:
nit, but id: should try and match the compat

id: nordic,nrf51-flash-controller